lib/YaST/DnsServer/DnsServerController.pm

```diff
@@ -0,0 +1,148 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces actions Dns Server Settings Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::DnsServer::DnsServerController;
+use strict;
+use warnings;
+use testapi;
```

Comment:
`testapi` is not needed.

Reply:
I will reserve this for later debug, at last will remove the testapi and save_screenshot in all the files if not needed any more.